app/metadata/options.tsx

```diff
@@ -4,8 +4,6 @@ export const workflowStatusOptions = [
   { value: 'pr', label: 'PR' },
   { value: 'prFailed', label: 'PR Failed' },
   { value: 'planned', label: 'Planned' },
-  { value: 'planFailed', label: 'Plan Failed' },
-  { value: 'approved', label: 'Approved' },
```

Comment:
What is the 'approved' status used for before?

Reply:
actually, we never used `approved` status before.
we added it when initially came up with the list of status and we thought we are going to use it for BCeID approved status; we have a separate field for it.